code/_onclick/hud/screen_objects.dm

```diff
@@ -320,8 +320,7 @@
 				if(R.module)
 					R.hud_used.toggle_show_robot_modules()
 					return 1
-				else
-					to_chat(R, "You haven't selected a module yet.")
+				to_chat(R, "You haven't selected a module yet.")
```

Comment:
Unable to easily confirm equivalent control flow

Reply:
`if(X) ... return else Y` is equivalent to `if(X) ... return; Y`

Reply:
Unless you meant to put this on the internals check section.

Reply:
This entire section looks fine to me, the logic has only been tabbed back.